lib/ansible/module_utils/csharp/Ansible.Basic.cs

```diff
@@ -374,7 +378,7 @@ public void LogEvent(string message, EventLogEntryType logEntryType = EventLogEn
             }
             if (sanitise)
                 message = (string)RemoveNoLogValues(message, noLogValues);
-            message = String.Format("{0} - {1}", ModuleName, message);
+            message = String.Format("{0} {1}- {2}", ModuleName, TargetLogInfo, message);
```

Comment:
You probably want to add a space between the `-`.
```suggestion
            message = String.Format("{0} {1} - {2}", ModuleName, TargetLogInfo, message);
```

Reply:
i didn't want to distrub existing lines with the extra space, I could always add it to targetloginfo itself?

Reply:
Ah ok then you'll have to conditionally add it. Something like

```csharp
if (String.IsNullOrWhiteSpace(TargetLogInfo))
{
    message = String.Format("{0} - {1}", ModuleName, message);
}
else
{
    message = String.Format("{0} {1} - {2}", ModuleName, TargetLogInfo, message);
}
```

We can do something more efficient with StringBuilder but it'll probably require a few more changes that aren't really needed for now.